File: 1975-minimum-distance-to-the-target-element/minimum-distance-to-the-target-element.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        n = len(nums)
        result = float("inf")
        for i in range(n):
            if result <= abs(i - start):
                break
            if nums[i] == target:
                result = min(result, abs(i - start))
        return result

File: 1975-minimum-distance-to-the-target-element/minimum-distance-to-the-target-element.cpp
class Solution {
public:
    int getMinDistance(vector<int>& nums, int target, int start) {
        int n = nums.size();
        int result = INT_MAX;

        for (int i = 0; i < n && result > abs(i - start); i++) {
            if (nums[i] == target) {
                result = min(result, abs(i - start));
            }
        }
        return result;
    }
};